Source/Fields/LMCCylindricalCavity.hh

```diff
@@ -56,6 +57,7 @@ namespace locust
             virtual void CheckNormalization(int nModes);
             virtual void PrintModeMaps(int nModes, bool bTE, double zSlice);
             virtual std::vector<double> GetFieldAtProbe(int l, int m, int n, bool includeOtherPols, std::vector<double> tKassParticleXP);
+            virtual std::vector<double> GetFieldAtProbe(int l, int m, int n, bool includeOtherPols, std::vector<double> tKassParticleXP, bool teMode);
```

Comment:
Can these two GetFieldAtProbe() functions be consolidated into just one function, or do they need to be overloaded?

Reply:
I implemented it as overloaded functions originally in case there were some calls to the original versions in other parts of the code and I didn't want to break other functionality, but after some quick tests it looks like it's fine with only the newest version of GetFieldAtProbe() and GetNormalizedModeField(). I've pushed the changes for both functions to remove the extra definitions

Reply:
Thanks for checking all of that.